os.path/os/path.py

```diff
@@ -1,7 +1,38 @@
 import os
-
+import sys
 
 sep = "/"
+altsep = "\\"
+if (sys.platform == 'uefi') and (sys.implementation.name == 'micropython'):
+ sep = '\\'
+ altsep = '/'
+ import _re as re
+else: import ure as re
```

Comment:
Code style not consistent.

Reply:
I don't follow. What code style am I violating?

Reply:
Well, MicroPython uses black for formatting. But only for the micropython repository currently, not yet for micropython-lib. In any case: all statements should go on their own line for consistency and readability